src/components/Summary/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { List } from 'antd';
+
+import { Container, ListItem, Profit, Footer, Content } from './styles.js';
+
+export default function Summary() {
+    // TODO: request from backend
+    const assets = [{
+        assetName: 'ITUB3',
+        averagePrice: 20,
+        sellPrice: 30,
+        totalSold: 10
```

Comment:
Some sugestions of variable names due to some read I took about stocks:

- stockSymbol (for the "code" of a stock);

- sellingPrice (for the price of a given sell order);
- quantityTraded or quantitySold (for the number of stocks in the order, once "total" may be misunderstood as the total amount or some other thing).

Reply:
It's done